README.md

```diff
@@ -71,11 +78,11 @@ npx expo run:ios
 # clearing the build cache
 npx expo run:ios --no-build-cache
 
-# build and run on Android connected devices
+# or build and run on Android connected devices
 npx expo run:android
 # clearing the build cache
 npx expo run:android --no-build-cache
 
-# just start the local server
+# and then just start the local server
```

Comment:
'and then' implies this is the next step, but the previous `npx expo run*` commands also run the server but just initialize the simulator, etc.